### ISSUE TITLE
Clean up examples/ directory

### ISSUE THREAD

**Issue by mattdangerw** [keras-team/keras-nlp#573]
There are a few minor cleanups we should do in the `examples/` directory for the 0.4 release.

 - We should remove the base readme https://github.com/keras-team/keras-nlp/blob/master/examples/README.md, it is quite out of date now that our models have moved inside the library.
 - We should remove [bert_glue_finetune.py](https://github.com/keras-team/keras-nlp/blob/master/examples/README.md). The new glue finetuning example is now much more fully featured.
 - We should rename the bert example to reflect it is now about pretraining specifically. Something like renaming `bert/` to `bert_pretraining/`, `bert_train.py` -> `bert_pretrain.py`, `bert_preprocess.py` -> `bert_create_pretraining_data.py`.

**Comment by chenmoneygithub** [comment]
I will handle this BERT pretraining later, it needs a few more works, e.g., using the MLMMaskGenerator to generate masks in our preprocessing.

**Comment by ADITYADAS1999** [comment]
> There are a few minor cleanups we should do in the `examples/` directory for the 0.4 release.
> 
> * We should remove the base readme https://github.com/keras-team/keras-nlp/blob/master/examples/README.md, it is quite out of date now that our models have moved inside the library.
> * We should remove [bert_glue_finetune.py](https://github.com/keras-team/keras-nlp/blob/master/examples/README.md). The new glue finetuning example is now much more fully featured.
> * We should rename the bert example to reflect it is now about pretraining specifically. Something like renaming `bert/` to `bert_pretraining/`, `bert_train.py` -> `bert_pretrained.py`, `bert_preprocess.py` -> `bert_create_pretraining_data.py`.

hey [mattdangerw](https://github.com/mattdangerw) this issue is open for work? Can I go through this 😃

**Comment by chenmoneygithub** [comment]
@ADITYADAS1999 thank you! Contributions are definitely welcome for this issue!
